dot/types/grandpa_test.go

```diff
@@ -64,21 +170,20 @@ func TestGrandpaAuthoritiesRawToAuthorities(t *testing.T) {
 		{Key: authB, ID: 1},
 	}
 
-	enc, err := scale.Marshal(auths)
-	require.NoError(t, err)
-	require.Equal(t, exp, enc)
+	encoding := scale.MustMarshal(auths)
+	require.Equal(t, expectedEncoding, encoding)
 
-	dec := []GrandpaAuthoritiesRaw{}
-	err = scale.Unmarshal(enc, &dec)
+	var grandpaAuthoritiesRaw []GrandpaAuthoritiesRaw
+	err := scale.Unmarshal(encoding, &grandpaAuthoritiesRaw)
 	require.NoError(t, err)
-	require.Equal(t, auths, dec)
+	require.Equal(t, auths, grandpaAuthoritiesRaw)
 
-	authoritys, err := GrandpaAuthoritiesRawToAuthorities(dec)
+	authorities, err := GrandpaAuthoritiesRawToAuthorities(grandpaAuthoritiesRaw)
 	require.NoError(t, err)
-	require.Equal(t, auths[0].ID, authoritys[0].Weight)
+	require.Equal(t, auths[0].ID, authorities[0].Weight)
 
-	a := Authority{}
-	err = a.FromRawEd25519(dec[1])
+	authority := Authority{}
+	err = authority.FromRawEd25519(grandpaAuthoritiesRaw[1])
 	require.NoError(t, err)
-	require.Equal(t, a, authoritys[1])
+	require.Equal(t, authority, authorities[1])
 }
```

Comment:
Unnecessary renaming, change it back please

Reply:
IMO it is a necessary renaming since it is fixing the typo `authoritys -> authorities` as well as making the variable name `a` more clear with `authority`